### PR TITLE
Use a keyword argument list for options here

### PR DESCRIPTION
**Pull request by nathanl**
Basedon @aaronrenner's suggestion on https://github.com/keathley/wallaby/pull/313

@aaronrenner I'm not sure how to update the typespec - feel like helping?

### CODE REVIEW

File: lib/wallaby/browser.ex [hunk 1]
```diff
@@ -200,16 +200,16 @@ defmodule Wallaby.Browser do
...
   @spec take_screenshot(parent) :: parent
 
-  def take_screenshot(%{driver: driver} = screenshotable, opts \\ %{}) do
+  def take_screenshot(%{driver: driver} = screenshotable, opts \\ []) do
     image_data =
       screenshotable
       |> driver.take_screenshot
 
-    path = path_for_screenshot(%{name: opts[:name]})
+    path = path_for_screenshot(name: opts[:name])
```
Comment (aaronrenner) [inline lib/wallaby/browser.ex]: What do you think about writing it like this?

```elixir
name = Keyword.get(opts, :name, :erlang.system_time)
path = path_for_screenshot(name)
```

That should simplify `path_for_screenshot/1` to

```elixir
defp path_for_screenshot(name) do
  File.mkdir_p!(screenshot_dir())
  "#{screenshot_dir()}/#{name}.png"
end

File: lib/wallaby/browser.ex [hunk 2]
```diff
@@ -200,16 +200,16 @@ defmodule Wallaby.Browser do
   Screenshots are saved to a "screenshots" directory in the same directory the
   tests are run in.
 
-  Pass %{name: "some_name"} to specify the file name. Defaults to a timestamp.
+  Pass `[{:name, "some_name"}]` to specify the file name. Defaults to a timestamp.
   """
   @spec take_screenshot(parent) :: parent
```
Comment (aaronrenner) [inline lib/wallaby/browser.ex]: You could probably update the typespec like this:

```elixir
@type take_screenshot_opt :: {:name, String.t}

@spec take_screenshot(parent, [take_screenshot_opt}]) :: parent
```

File: integration_test/cases/browser/screenshot_test.exs [hunk 3]
```diff
@@ -61,7 +61,7 @@ defmodule Wallaby.Integration.Browser.ScreenshotTest do
 
     [screenshot_path] =
       page
-      |> take_screenshot(%{name: "some_page"})
+      |> take_screenshot(name: "some_page")
```
Comment (aaronrenner) [inline integration_test/cases/browser/screenshot_test.exs]: It seems a lot nicer to type, now. 👍

**Comment by nathanl** [comment]
@aaronrenner See update - thanks for your help!

**Pull request merged**